src/Prima/Pyxis/Modal.elm

```diff
@@ -1276,7 +1335,7 @@ Icon for closing modal
 headerCloseButton : msg -> Html msg
 headerCloseButton closeEvent =
     Html.i
-        [ HtmlAttributes.class "icon icon-close modal__close"
+        [ HtmlAttributes.class "modal__header__close icon icon-close"
```

Comment:
Quindi l'icona close è fatta con `icomoon` o con il `css`?

Reply:
In css, ho rimosso le classi icon